solc/CommandLineInterface.cpp

```diff
@@ -839,6 +841,7 @@ Allowed options)").c_str(),
 		)
 		(g_argColor.c_str(), "Force colored output.")
 		(g_argNoColor.c_str(), "Explicitly disable colored output, disabling terminal auto-detection.")
+		(g_argErrorIds.c_str(), "Output error codes, even when --standard-json is not specified.")
```

Comment:
I think the standard-json-part might be confusing - we also do not talk about than for e.g. "Force colored output".

Reply:
OK as `--standard-json` already says
> Switch to Standard JSON input / output mode, **ignoring all options**.